From 05d8321118f4a57b7a7c4223a7baf9cca0974dec Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Wed, 30 Jan 2013 18:44:20 +0000
Subject: [PATCH] configs: vexpress: Enable LEDs

To provide access to LEDs enable

  CONFIG_LEDS_CLASS=y
  CONFIG_LEDS_GPIO=y

(LED's are available from user side under under /sys/class/leds)

To use the motherboard LEDs to show CPU activity...

  CONFIG_LEDS_TRIGGERS=y
  CONFIG_LEDS_TRIGGER_HEARTBEAT=y
  CONFIG_LEDS_TRIGGER_CPU=y

Signed-off-by: Jon Medhurst <tixy@linaro.org>
(cherry picked from commit fe44c85751532df4169d80949b0ca4680d6dee6f)
Signed-off-by: Alex Shi <alex.shi@linaro.org>
---
 linaro/configs/vexpress.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/linaro/configs/vexpress.conf b/linaro/configs/vexpress.conf
index 8c1c909..6576492 100644
--- a/linaro/configs/vexpress.conf
+++ b/linaro/configs/vexpress.conf
@@ -39,3 +39,7 @@ CONFIG_REGULATOR=y
 CONFIG_REGULATOR_VEXPRESS=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
+CONFIG_LEDS_GPIO=y
+CONFIG_LEDS_TRIGGERS=y
+CONFIG_LEDS_TRIGGER_HEARTBEAT=y
+CONFIG_LEDS_TRIGGER_CPU=y
-- 
1.8.0

